### ISSUE TITLE
Soundcloud playlist failure - 400 error

### ISSUE THREAD

**Issue by aikaterna** [sedmelluq/lavaplayer#284]
Using Lavaplayer 1.3.38 in Lavalink.

Any Soundcloud set or playlist that has greater than 50 songs in the playlist will error out.

```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Loading playlist from SoundCloud failed.
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudPlaylistLoader.loadFromSet(DefaultSoundCloudPlaylistLoader.java:83) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudPlaylistLoader.load(DefaultSoundCloudPlaylistLoader.java:61) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioSourceManager.loadItem(SoundCloudAudioSourceManager.java:119) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218) ~[lavaplayer-1.3.38.jar!/:na]
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.io.IOException: Invalid status code for track list response: 400
        at com.sedmelluq.discord.lavaplayer.tools.io.HttpClientTools.assertSuccessWithContent(HttpClientTools.java:153) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudPlaylistLoader.loadPlaylistTracks(DefaultSoundCloudPlaylistLoader.java:101) ~[lavaplayer-1.3.38.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudPlaylistLoader.loadFromSet(DefaultSoundCloudPlaylistLoader.java:78) ~[lavaplayer-1.3.38.jar!/:na]
        ... 9 common frames omitted
```

Examples:

Works:
https://soundcloud.com/aikaterna/sets/p5-1 (3 songs)
https://soundcloud.com/setoftheday/sets/releases (6 songs)
https://soundcloud.com/tycho/sets/rootmusic-bandpage (15 songs)
https://soundcloud.com/runthetrap/sets/run-the-traps-top-50-trap (49 songs)
https://soundcloud.com/highonmusic1/sets/chill-songs-2019 (50 songs)

Throws traceback:
https://soundcloud.com/musicis4lovers/sets/free-downloads (51 songs)
https://soundcloud.com/soaverecords/sets/house-remixes (59 songs)
https://soundcloud.com/daniel-galabov-769051185/sets/lofi-triphop-1 (over 450 songs)

**Comment by Devoxin** [comment]
Making a note that this is caused by requests to the `/tracks` endpoint with more than 50 track IDs passed to the `ids` query parameter.

Potential solution: Split IDs into lists of 50. Request each "segment" and aggregate into one list of tracks.

**Comment by aikaterna** [comment]
This is resolved in 1.3.47.
